runtime/vm/jfr.cpp

```diff
@@ -739,8 +842,16 @@ jfrSamplingThreadProc(void *entryArg)
 		omrthread_monitor_enter(vm->jfrSamplerMutex);
 		vm->jfrSamplerState = J9JFR_SAMPLER_STATE_RUNNING;
 		omrthread_monitor_notify_all(vm->jfrSamplerMutex);
+		UDATA count = 0;
 		while (J9JFR_SAMPLER_STATE_STOP != vm->jfrSamplerState) {
 			J9SignalAsyncEvent(vm, NULL, vm->jfrAsyncKey);
+			if (count % 100 == 0) { // 1000ms
+				jfrCPULoad(currentThread);
+			}
+			if (count % 1000 == 0) { // 10s
```

Comment:
Missing parentheses, constants go on the left of `==`:
```c
			if (0 == (count % 100)) { // 1000ms
```
By the way, how do you conclude that 100 counts is 1000ms?

Reply:
`J9JFR_SAMPLING_RATE` is defined to be 10 so 100 counts is 1000ms